pkgs/development/python-modules/pytest-json-report/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pytest-metadata
+, pytest-xdist
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-json-report";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "numirias";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-OS9ASUp9iJ12Ovr931RQU/DHEAXqbgcRMCBP4h+GAhk=";
+  };
+
+  buildInputs = [
```

Comment:
why buildInputs for pytest?

Reply:
Was requested several times the past to put it in `buildInputs` instead of `propagatedBuildInputs`. Was, as far was I remember, required to break circular dependencies with pytest.

It could be that my knowledge is outdated.